developer-docs/bolt-api-servers.md

```diff
@@ -165,9 +166,18 @@ The metadata object is optional, and contains metadata about the task being run.
 - `parameters`: Object, *optional* - A JSON object whose keys are parameter names, and whose values are JSON objects with 2 keys:
     - `description`: String, *optional* - The parameter description.
     - `type`: String, *optional* - The type the parameter should accept.
+    - `sensitive`: Boolean, *optional* - Whether the task runner should treat the parameter value as sensitive
+    - `input_method`: String, *optional* - What input method should be used to pass params to task (stdin, environment, powershell)
 
 #### Files
-# TODO
+The files array contains details about the files the task needs as well as how to get them. Array items should be objects with the following keys:
```

Comment:
I think include that it's required? Should we just add an italicized `*required*` next to that and name? Or is it obvious from examples + other docs that it's required?